vpn_slice/provider.py

```diff
@@ -143,3 +143,18 @@ def prepare_tunnel(self):
         Base class behavior is to do nothing.
 
         """
+
+class DomainDNSProvider:
```

Comment:
I might call this the `SplitDNSProvider` and change the related names throughout.

Reply:
done :) actually had that initially